docs/source/_extensions/_static/accordian.css

```
@@ -0,0 +1,33 @@
+/* Accordians */
```

```suggestion
/* Accordions */
```
Probably also adjust the filename